### DISCUSSION TITLE
Dynamic Groups by Age Filter

### DISCUSSION THREAD

**Discussion by YannickGutmann** [liedekef/events-made-easy#688] [Ideas]
Would it be possible to add an additional filter for Dynamic Groups, which allows filtering members based on a minimum age? This would be useful for events that require a minimum age to attend.

Thank you in advance!

**Comment by liedekef** [comment]
This changeset should help (untested, but should be ok): https://github.com/liedekef/events-made-easy/commit/23debd0b1a4ac0e7aacc0a8b32b070384fa19c97
